javascript/app.js

```diff
@@ -2,15 +2,215 @@ const menuBars = document.getElementById('menu-bars');
 const mobileMenu = document.getElementById('mobile-menu');
 const closeMenuBtn = document.getElementById('close-menu-btn');
 const mobileLinks = document.getElementById('mobile-links');
+const workModal = document.getElementById('work-modal');
 
-function openMenuHandler() {
+const preventDefaultHandler = (e) => {
+  e.preventDefault();
+};
+
+const openMenuHandler = () => {
   mobileMenu.classList.replace('d-none', 'd-flex');
-}
+  document.body.classList.add('stop-scrolling');
+  document.body.addEventListener('touchmove', preventDefaultHandler);
+};
 
 function closeMenuHandler() {
   mobileMenu.classList.replace('d-flex', 'd-none');
+  document.body.classList.remove('stop-scrolling');
+  document.body.removeEventListener('touchmove', preventDefaultHandler);
 }
 
 menuBars.addEventListener('click', openMenuHandler);
 closeMenuBtn.addEventListener('click', closeMenuHandler);
-mobileLinks.addEventListener('click', closeMenuHandler);
\ No newline at end of file
+mobileLinks.addEventListener('click', closeMenuHandler);
+
+const worksArray = [
+  {
+    id: 0,
+    title: 'Tonic',
+    experiences: ['Canopy', 'Back End Dev', '2015'],
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+    technologies: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    urlMobile: '/images/work_1.jpg',
+    urlDesktop: '/images/work_1_lg.png',
+    urlModal: '/images/work_modal.png',
+    alt: 'work_tonic_daily_personalized_reads',
+    live: 'https://hafedefheij.github.io',
+    source: 'https://hafedefheij.github.io/mobile-portfolio/',
+  },
```

Comment:
- [ ] According to the previous reviewer, stated that the links to your card images are broken and I notice you didn't fix them, but currently the card's image links are still broken for both the main page and modal pop-up in your desktop and mobile versions respectively.

 - Kindly check the image path and fix them accordingly

Reply:
@brainconnect93 about the links I think I fixed them from the first review, the problem before was the missing picture  they working on my machine,  can you please guide me in that